concordium-node/src/consensus_ffi/ffi.rs

```diff
@@ -709,14 +714,39 @@ impl ConsensusContainer {
         ))
     }
 
-    pub fn in_baking_committee(&self) -> ConsensusIsInBakingCommitteeResponse {
-        wrap_c_committee_call!(self, |consensus| bakerIdBestBlock(consensus))
+    /// Gets baker status of the node along with the baker ID registered in the
+    /// baker credentials used, if available.
+    pub fn in_baking_committee(&self) -> (ConsensusIsInBakingCommitteeResponse, Option<u64>) {
+        let consensus = self.consensus.load(Ordering::SeqCst);
+        let mut baker_id: u64 = 0;
+        let mut has_baker_id: u8 = 0;
+
+        let result = unsafe { bakerStatusBestBlock(consensus, &mut baker_id, &mut has_baker_id) };
+
+        let status =
+            ConsensusIsInBakingCommitteeResponse::try_from(result).unwrap_or_else(|code| {
+                unreachable!("Unknown Consensus Baking Committee FFI return code: {}", code)
```

Comment:
THis is a minor nitpick, but if this case ever happens, the way the errors are phrased it will print something like
```
Unknown Consensus Baking Committee FFI return code: Unsupported FFI return code for committee status ({})
```

Reply:
Good catch, changed it to something less redundant.